staging/src/kubevirt.io/api/core/v1/types.go

```diff
@@ -1477,6 +1484,15 @@ type VirtualMachineStateChangeRequest struct {
 	UID *types.UID `json:"uid,omitempty" optional:"true" protobuf:"bytes,5,opt,name=uid,casttype=k8s.io/kubernetes/pkg/types.UID"`
 }
 
+type VirtualMachineInterfaceRequest struct {
+	// AddInterfaceOptions when set indicates a volume should be added. The details
+	// within this field specify how to add the volume
+	AddInterfaceOptions *AddInterfaceOptions `json:"addInterfaceOptions,omitempty" optional:"true"`
+	// RemoveInterfaceOptions when set indicates a volume should be removed. The details
+	// within this field specify how to add the volume
+	RemoveInterfaceOptions *RemoveInterfaceOptions `json:"removeInterfaceOptions,omitempty" optional:"true"`
```

Comment:
The PR is for hotplug only, isn't it?

Reply:
same explanation as https://github.com/kubevirt/kubevirt/pull/6852#discussion_r1070970169

Reply:
Done.